airbyte-integrations/connectors/source-hubspot/source_hubspot/client.py

```diff
@@ -81,8 +81,10 @@ def streams(self) -> Iterator[AirbyteStream]:
         for stream in super().streams:
             properties = self._apis[stream.name].properties
             if properties:
-                stream.json_schema["properties"]["properties"] = {"type": "object", "properties": properties}
-            stream.default_cursor_field = [self._apis[stream.name].updated_at_field]
+                stream.json_schema["properties"]["properties"] = {
+                    "type": "object", "properties": properties}
+            stream.default_cursor_field = [
+                self._apis[stream.name].updated_at_field]
```

Comment:
This should be formatted like that. Please check out the `./gradlew format` command https://docs.airbyte.io/contributing-to-airbyte/developing-locally#run-formatting-automation-tests
```suggestion
                stream.json_schema["properties"]["properties"] = {"type": "object", "properties": properties}
                stream.default_cursor_field = [self._apis[stream.name].updated_at_field]
```